js/explore/largeRepos/pie_activityLines.js

```diff
@@ -0,0 +1,156 @@
+/* Creates pie chart visualization for webpage */
+function draw_pie_lines(areaID) {
+    // load data file, process data, and draw visualization
+    var url = ghDataDir + '/labRepos_ActivityLines.json';
+    var files = [url];
+    Promise.all(files.map(url => d3.json(url))).then(response => {
+            var data = reformatData(response[0]);
+            drawGraph(data, areaID);
+        });
+
+    // Draw graph from data
+    function drawGraph(data, areaID) {
+        var graphHeader = 'Lines Added This Year';
```

Comment:
The title for "Lines Added" seems a bit misleading as far as new lines vs edited lines....
Perhaps "Line Changes" would be a good alternative? (I'm having a hard time coming up with another good option that fits without shrinking text, unless we want to leave out "This Year".)

```suggestion
        var graphHeader = 'Line Changes This Year';
```

Reply:
To me, changes would denote deletions and additions, though I get what you mean about not being able to fit a better title in the space. We could add "this year" to the tip and take it out in the title?

Reply:
Something like, "Line edits and additions"